### DISCUSSION TITLE
Move `useTemplate` and `useSimplifiedLogicalExpression` to `style` group

### DISCUSSION THREAD

**Discussion by MichaReiser** [rome/tools#3461] [Suggestions]
## Question

To move the following rules to the stylistic lint group:

* `useTemplate`: Recommends using template literals over string concatenation. In my view, this is more a stylistic preference than preventing bugs (both variants can become difficult to read when they contain a lot or specially complicated placeholders). 
* `useSimplifiedExpression`: Helps to simplify logical expression. I can see the point how complex expression may increase the likeliness of bugs but sometimes comes down to how someone prefers to group logical expressions (arguably, it may then be recommended to create temporary variables).		

## Recommendation
Rules to be in stylistic group

**Comment by ematipico** [comment]
We could follow `clippy` lead, and move `useSimplifiedExpression` under a new group called `complexity`: 

> code that does something simple but in a complex way

If you check their Complexity group here https://rust-lang.github.io/rust-clippy/master/index.html (you need to select the filter), there's a bunch of rules around binary expressions.

**Reply by MichaReiser** [reply]
I like that!

**Comment by ematipico** [comment]
This proposal is implemented in #3471
